css/index.css

```diff
@@ -1,3 +1,172 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* Desktop CSS here! */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
 
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+body .container {
+	margin: 0 auto;
+	max-width: 880px;
+}
+
+.navigation {
+	  display: flex;
+		flex-direction: row;
+		padding-top: 15px;
+		padding-bottom: 15px;
+		justify-content: space-between;
+		align-items: center;
+}
+
+nav a {
+	color: grey;
+	text-decoration: none;
+	height: inherit;
+	margin-right: 50px;
+}
+
+nav a:first-child {
+	padding-left: 0;
+}
+
+.logo {
+	float: right;
+	margin-left: 10px;
+}
+
+header {
+  margin-top: 45px;
+  margin-bottom: 50px;
+}
+
+.header {
+	display: inline-block;
+  vertical-align: top;
+  font-size: 40px;
+  margin-left: 60px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.header-img {
+	float: right;
+	display: inline-block;
+}
+
+hr {
+  border: 1px solid black;
+}
+
+.header button {
+	width: 125px;
+	height: 25px;
+	font-size: 12px;
+	display: inline-block;
+  margin-left: 0;
+	border-color: #585858;
+  border-width: 1px;
+	box-shadow: none;
+	background-color: white;
+}
+
+.first-half,
+.second-half {
+  display: inline-block;
+  margin: 15px 0px;
+}
+
+.first-half-content,
+.second-half-content {
+	display: inline-block;
+  margin-right: 15px;
+}
+
+.first-half-content {
+	vertical-align: top;
+	width: 45.5%;
+}
+
+.second-half-content {
+	width: 30%;
+}
+
```

Comment:
You won't need to set `width` for these elements if they're place inside flexboxes (better to change the flexbox properties of their container instead so that they will be centered automatically, as opposed to defining `width` values).